include/functions.h

```diff
@@ -831,7 +831,7 @@ void func_800645A0(GlobalContext* globalCtx, CutsceneContext* csCtx);
 void Cutscene_HandleEntranceTriggers(GlobalContext* globalCtx);
 void Cutscene_HandleConditionalTriggers(GlobalContext* globalCtx);
 void Cutscene_SetSegment(GlobalContext* globalCtx, void* segment);
-void* MemCopy(void* dest, void* src, s32 size);
+void* MemCpy(void* dest, const void* src, s32 size);
```

Comment:
since with the capitalization it's already not `memcpy`, is there really a point to removing the `o`
This may just bring more confusion between `memcpy` and `MemCpy`, not that the `o` would help much but it's still something (at least both functions are conformant `memcpy`s, but one is in code so may not be used in boot so it's not entirely irrelevant to prevent confusion)

Reply:
Ideally these get a proper prefix anyway. Unfortunately there's very little context for how they're used: it's essentially "the one used outside `boot`", and the `MemSet` in here isn't used at all. It's totally gone in MM, and its duties reassigned to `bcopy` or `__osMemcpy`, so practically the advice should just be "don't bother using this one".